### ISSUE TITLE
More clearance height data needed

### ISSUE THREAD

**Issue by PetersonGIS** [terranodo/tegola-openseamap#20]
It looks like we also need to style seamark_misc_lines and seamark_misc_points where type='pipeline_overhead' and seamark_cable_overhead the way that we styled bridge points (as per https://github.com/terranodo/tegola-openseamap/issues/13). 

They all essentially have the same symbol and need to be labeled with the clearance height. So we need to bring in the clearance_height data for these as well.

**Comment by PetersonGIS** [comment]
This is also needed for bridge_areas and bridge_lines.

**Comment by erictheise** [comment]
I added the `clearance_` fields to `bridge_lines` and `bridge_areas` at the same time I added it to `bridge_points`. I'm seeing `clearance_` data for a good portion of the 396 `bridge_lines` and 20 `bridge_areas` we currently have in production.

**Comment by PetersonGIS** [comment]
Ok. Sounds good, I must have been looking at some bridge lines and bridge areas that just didn't have clearance stats. Is it possible to get the clearance numbers for misc lines and misc points where type=="pipeline_overhead" and where type=="seamark_cable_overhead"?

**Comment by erictheise** [comment]
I find `vertical_clearance` (3 rows) and `clearance_height_safe` (6 rows) data in the 111 row `cable_overhead` table. But I was only treating `cable_overhead` as lines and now I see there are points (800+) as well. So I'll import into two tables, `_lines` and `_points`.

I'll break `pipeline_overhead` out of `misc` and capture the clearance numbers. Do you know offhand what the fields are called? They're not spelled out in the schema pages.

**Comment by erictheise** [comment]
After some research we determined that `pipeline_overhead` objects, counterintuitively, do not have clearance data.

18caf0b2 has a new representation of `cable_overhead` with `cable_overhead_lines` and `cable_overhead_points`. I did adjust `nautical_active.json` to look for those but made no changes to styles.

Our planet data shows only 2 rows in `cable_overhead_points` having `clearance_height_safe` and none having `vertical_clearance`.
